commit a5504cf18a25ab6cbb47da04103048b240c7edc1 (HEAD)
Author: Simon Zeni <simon@bl4ckb0ne.ca>
Date:   Wed Feb 2 13:54:55 2022 -0500

    generated_header: add missing stddef include

note(void-packages): this patch is necessary to fix a compile error
in musl targets

--- a/libsurvive-1.0/libs/cnkalman/include/cnkalman/generated_header.h
+++ b/libsurvive-1.0/libs/cnkalman/include/cnkalman/generated_header.h
@@ -1,4 +1,5 @@
 #pragma once
+#include <stddef.h>
 #include <math.h>
 static inline double __cnkalman_safe_sqrt(double x) { return x > 0 ? sqrt(x) : 0; }
 #define sqrt __cnkalman_safe_sqrt

